docs/concepts/package-installation-process.md

```diff
@@ -47,3 +47,76 @@ The general process is as follows:
 7. (Visual Studio only) Display the package's readme file, if available, in a Visual Studio window.
 
 Enjoy your productive coding with NuGet packages!
+
+## Validating source of installed packages
```

Comment:
These headers become HTML anchors, so I feel like it's important to try to get them  right the first time, since changing them in the future will break links.

If you have any suggestsions for improvements to any of these headers, please let me know.

Reply:
How about `Auditing`?

Reply:
+1 for Auditing as I think it's a great word for this.

Reply:
We synced up offline, I think @JonDouglas's suggestion for a separate doc makes sense. 

I might've given conflicting feedback in the past, but the sheer size of content we're adding makes me want to reconsider it.